packages/msi/oh-my-posh.dsc.resource.json

```
@@ -0,0 +1,59 @@
+{
+  "$schema": "https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2024/04/bundled/resource/manifest.json",
```

With the official release of DSC, this is no longer a valid/recognized schema URI. Strongly recommend using this URI for forward compatibility:

```suggestion
  "$schema": "https://aka.ms/dsc/schemas/v3/bundled/resource/manifest.json",
```

If you're concerned about editing your schema in VS Code, I recommend you use the enhanced authoring schema (VS Code doesn't recognize canonically bundled schemas yet, as support for 2020-12 is only partially implemented):

```suggestion
  "$schema": "https://aka.ms/dsc/schemas/v3/bundled/resource/manifest.vscode.json",
```

For more information about schema URIs and DSC, see [DSC JSON Schema URIs][01].

[01]: https://learn.microsoft.com/powershell/dsc/reference/schemas/schema-uris

---

Is there a VS Code issue to track for that support?

---

Here you go: https://github.com/microsoft/vscode/issues/165219. It will also be documented in a known issue [document](https://github.com/PowerShell/DSC/pull/926).